tests/integration_tests/core/test_cli_response.py

```diff
@@ -0,0 +1,41 @@
+import subprocess
+
+VALIDATE_COMMAND = "rasa data validate --data data/test/test_integration \
+\-d data/test/test_integration/domain.yml"
```

Comment:
in python, it's usually preferred to not use an escape character for strings, because there are other ways to do this more conveniently. For instance you can break your string into multiple lines like this (notice there is no `,` so it's not a tuple)
```suggestion
VALIDATE_COMMAND = (
    "rasa data validate --data data/test/test_integration "
    "-d data/test/test_integration/domain.yml"
)
```